go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -216,7 +216,7 @@ func (c *Controller) handleErr(err error, key interface{}) {
 
 	ns, name, keyErr := cache.SplitMetaNamespaceKey(key.(string))
 	if keyErr != nil {
-		klog.ErrorS(err, "Failed to split meta namespace cache key", "key", key)
+		klog.ErrorS(err, "failed to split meta namespace cache key", "key", key)
```

Comment:
Why is this lowercase?

Reply:
@trozet the regex in verify-gofmt.sh is so annoying! i have been playing with https://regex101.com/ for sometime.. can't get handle of the greedy match.. 

![image](https://user-images.githubusercontent.com/4494709/146975634-30328b93-4ea5-47a6-89ce-70f9e8186dfd.png)